src/main/java/study/stringcalculator/StringAddCalculator.java

```diff
@@ -12,9 +12,7 @@ public static int splitAndSum(String text) {
         if (isNullOrEmpty(text)) {
             return 0;
         }
-        String separator = getSeparator(text);
-        String numberString = getNumberText(text, separator);
-        String[] stringNumbers = splitNumbers(numberString, separator);
+        String[] stringNumbers = splitText(text);
         int[] numbers = convertToIntegers(stringNumbers);
         return sumNumbers(numbers);
```

Comment:
```suggestion
return sumNumbers(convertToIntegers(splitText(text)));
```
굳이 로컬 변수가 필요 없다면 위와 같이 구현하는 것은 어떨까?
위와 같이 메서드 체인 방식으로 구현하는 것도 익숙해지기 위해 의식적으로 연습해 볼 것을 추천

Reply:
강의에서도 말씀해주셨던 내용인데 놓쳤었네요